src/lib/Util/adapters/bbc-kaldi/group-words-by-speakers.js

```diff
@@ -0,0 +1,122 @@
+/**
```

Comment:
This is kind of a contained point but I've just noticed how wacky the file and folder naming is for this project. Half of the time, folders are capitalised (for react etc.) or camel cased. and then sometimes it's dashes.

😱

Reply:
lol, we'll address in https://github.com/bbc/react-transcript-editor/pull/122 :)